### PR TITLE
Bump up version to 0.1.3

### PR DESCRIPTION
**Pull request by SajidAlamQB**
Release a new minor version.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Release 0.1.3
+
+## Bug fixes and other changes
+* CLI command arguments on `kedro-telemetry` are now masked client-side, this allows for a more maintainable vocabulary.
```
Comment (lorenabalan) [inline RELEASE.md:4]: > this allows for a more maintainable vocabulary.

Not sure what that means but sure. 😂

**Review by lorenabalan — APPROVED** [review]
🚀 🚀

File: RELEASE.md (hunk 1 above)
Comment (SajidAlamQB) [reply RELEASE.md:4]: Stolen from @jiriklein [PR](https://github.com/quantumblacklabs/kedro-telemetry/pull/5) 😂

**Review by MerelTheisenQB — APPROVED** [review]
:shipit:

**Pull request merged**